src/test/ui/borrowck/borrow-tuple-fields.rs

```diff
@@ -30,12 +30,12 @@ fn main() {
 
     let mut x = Bar(1, 2);
     let a = &x.0;
-    let b = &mut x.0; //~ ERROR cannot borrow `x.0` as mutable because it is also borrowed as
+    let b = &mut x.0; //~ ERROR cannot borrow `x` as mutable because it is also borrowed as
```

Comment:
what's wrong with `x.0` here? Isn't the point of #99470 that we should not say `x.0` if it's an enum destructuring, not getting rid of the `.0`/`.1`/etc. altogether?

Reply:
I am going to fix this